### PR TITLE
Update internationalization.md to include the necessary import statement

### PR DESCRIPTION
**Pull request by valerianaGit**
Dev must import dependency for `import 'package:flutter_gen/gen_l10n/app_localizations.dart';` to be able to use app localizations.

_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by domesticmouse** [comment]
Hey @valerianaGit we can't land this PR until the CLA is signed. Thanks!

**Comment by valerianaGit** [comment]
@domesticmouse signed!

**Comment by domesticmouse** [comment]
I suspect this will fail the excepts test. The code you have changed is pulled from a backing `dart` file in the project referenced in the line above your change. You will need to make the change in both places.

File: src/ui/accessibility-and-internationalization/internationalization.md [hunk 1]
```diff
@@ -319,6 +319,7 @@ complete the following instructions:
 
    <?code-excerpt "gen_l10n_example/lib/main.dart (InternationalizedTitle)"?>
```
Comment (domesticmouse) [inline src/ui/accessibility-and-internationalization/internationalization.md:320]: You'll need to modify this code as well

**Review by domesticmouse — CHANGES_REQUESTED** [review]

**Comment by parlough** [comment]
Thanks so much for the PR!

Since the PR been sitting for a while with changes requested, I'm going to close it for now. However, if you want to revisit the PR, please feel free to reopen it or open a new one.

I will note, that this step is a continuation of the previous step which does highlight adding this import. This snippet is intended to just be a focused portion of the file. Is there some way we can make that more clear, either textually or visually?

Thanks again :)

**Pull request closed without merging**